src/ContentRepository90/Rules/FusionNodePathRector.php

```diff
@@ -30,14 +30,10 @@ public function refactorFileContent(string $fileContent): string
                 '// TODO 9.0 migration: Line %LINE: You may need to rewrite "VARIABLE.path" to Neos.Node.path(VARIABLE). We did not auto-apply this migration because we cannot be sure whether the variable is a Node.'
             )
             ->process(fn(string $eelExpression) => preg_replace(
-                '/(node|documentNode|site)\.property\\(\'_path\'\\)/',
+                '/q\((node|documentNode|site)\)\.property\\((\'|")_path(\'|")\\)/',
```

Comment:
Basically see above…

Too complicated, and what about `site`? People might be using any name instead of just `node`, `documentNode` or `site`.

```suggestion
                '/q\([^)]+\)\.property\\([\'"]_path[\'"]\)/',
```